src/components/tabs/tabs.component.js

```diff
@@ -356,6 +358,8 @@ Tabs.propTypes = {
     "no right side",
     "no sides",
   ]),
+  /** sets tabsHeaderWidth to TabHeader */
```

Comment:
```suggestion
  /** sets width to the tab headers. Can be any valid CSS string */
```

Reply:
The user has no visibility of the `TabHeader` component because it's an internal one